tool/src/js/type_generation/converter.rs

```diff
@@ -118,7 +118,7 @@ impl<'jsctx, 'tcx> TyGenContext<'jsctx, 'tcx> {
                             .formatter
                             .fmt_lifetime_edge_array(lt, lifetime_environment)
                             .into_owned(),
-                        _ => panic!("'static not implemented for JS2 backend"),
+                        _ => panic!("'static not supported for JS backend"),
```

Comment:
now that you have a `supports.static_slices`, you could fail during lowering

Reply:
Done, I've had to add a check to see if a method is disabled to support this for params, though.

Didn't do this for field types because there's already a check to see if structs are disabled. Not sure if we want to support disabling fields based on the backend though.